packages/flutter_test/lib/src/test_async_utils.dart

```diff
@@ -307,6 +307,7 @@ class TestAsyncUtils {
     do { // skip past frames that are from this class
       index += 1;
       assert(index < stack.length);
+      // ignore: unnecessary_null_checks
       lineMatch = getClassPattern.matchAsPrefix(stack[index])!;
       assert(lineMatch != null);
```

Comment:
```suggestion
      lineMatch = getClassPattern.matchAsPrefix(stack[index]);
      assert(lineMatch != null);
      lineMatch = lineMatch!;
```

Reply:
Hey @a14n 
thanks for the suggestion...
should I do this for 2nd ignore stuff as well?